src/validate-reducer/index.js

```diff
@@ -0,0 +1,23 @@
+export {validateReducer}
+
+const EXIT_EARLY = 'EXIT_EARLY'
+const CONTINUE = 'CONTINUE'
+
+validateReducer.EXIT_EARLY = EXIT_EARLY
+validateReducer.CONTINUE = CONTINUE
```

Comment:
Or you could look at the stats validator in Webpack validator for a real world example